### ISSUE TITLE
Do LHR rtx gpu limit throughput? 

### ISSUE THREAD

**Issue by 0xGitGuy** [johguse/profanity#55]
Does LHR gpu suffer from reduced speed compared to the NON-LHR of the same gpu model?

**Comment by its5Q** [comment]
No, LHR cards don't limit hashing performance, they only limit mining algorithms. For example, hashcat - a popular hash cracking software - has the same performance on LHR and non-LHR cards

**Comment by 0xGitGuy** [comment]
@its5Q Thank you!
